lib/archethic/mining/fee.ex

```diff
@@ -95,14 +86,13 @@ defmodule Archethic.Mining.Fee do
 
   defp do_calculate(
          uco_price_in_usd,
-         transaction_value,
          nb_bytes,
          nb_storage_nodes,
          nb_recipients
        ) do
     # TODO: determine the fee for smart contract execution
 
-    value_cost = fee_for_value(uco_price_in_usd, transaction_value)
+    value_cost = 0.01 / uco_price_in_usd
```

Comment:
I think you can rename it as "minimum fee"
You could also use a constant for the 0.01 , this will be easier to update.